.github/workflows/build-and-push-prover-server.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: Build and push Shielder-Prover-Server docker image (host app)
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'git ref: hash, branch, tag to build shielder-prover-server files from'
+        type: string
+        required: true
+
+jobs:
+  main:
+    name: Build Shielder Prover Server (host app)
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref }}
+          fetch-depth: 0
+
+      - name: Call action get-ref-properties
+        id: get-ref-properties
+        uses: Cardinal-Cryptography/github-actions/get-ref-properties@v7
+
+      - name: Login to Public Amazon ECR
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ vars.ECR_PUBLIC_HOST }}
+          username: ${{ secrets.AWS_MAINNET_ECR_CC_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_MAINNET_ECR_CC_ACCESS_KEY }}
+
+      - name: DOCKER | Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          version: v0.9.1
+
+      - name: Build and push docker image
+        id: build-image
+        uses: docker/build-push-action@v3
+        with:
+          context: tee
+          builder: ${{ steps.buildx.outputs.name }}
+          file: ./tee/docker/Dockerfile
+          push: true
+          # yamllint disable rule:line-length
+          tags: |
+            ${{ vars.ECR_CC_RES_PUBLIC_REGISTRY }}shielder-prover:${{ steps.get-ref-properties.outputs.sha }}
+            ${{ vars.ECR_PUBLIC_HOST }}shielder-prover:latest
```

Comment:
those both should be ECR_CC_RES_PUBLIC_REGISTRY

Reply:
Good catch!